indico/legacy/pdfinterface/latex.py

```diff
@@ -9,12 +9,12 @@
 import os
 import subprocess
 import tempfile
+from importlib.resources import files, as_file
```

Comment:
`files` is super generic, hence the linter error about shadowing below. maybe importing it `as importlib_res_files` would be better?

Reply:
Or `from importlib import resources as importlib_res` and using qualified names for `files` and `as_file`... I really don't understand what the importlib devs where thinking when creating module-level functions with such generic names that are really not nice to import w/o aliasing...